### DISCUSSION TITLE
File

### DISCUSSION THREAD

**Discussion by Odilon97** [dr5hn/countries-states-cities-database#661] [Q&A]
How i Can get this file ? 
I need it for my project ??? 
Country states city ....

**Comment by dr5hn** [comment]
https://github.com/dr5hn/countries-states-cities-database

You can download zip.
<img width="1033" alt="Screenshot 2023-08-11 at 4 01 12 PM" src="https://github.com/dr5hn/countries-states-cities-database/assets/6929121/461768cc-5cd6-453f-baf3-705106dd0ca2">
